### DISCUSSION TITLE
guide/chart/mermaid/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#303] [docs-comment]
# guide/chart/mermaid/

概述 主题支持在 文章中 嵌入由 Mermaid 。 该功能由 vuepress-plugin-md-enhance 提供支持。 配置 主题默认不启用该功能。 你需要在你的项目中安装 mermaid 库。 然后在 .vuepress/config.ts 配置文件中，启用该功能： 注 以下文档 Fork 自 vuepress-theme-hope, 遵循...

https://theme-plume.vuejs.press/guide/chart/mermaid/

**Comment by zhengchen-zzZ** [comment]
作者，按照这个文档我进行了mermaid的安装及配置，为什么装了没有生效

**Reply by pengzhanbo** [reply]
我在本地新建了项目，并安装了 mermaid 和 配置了 `plugins.markdownEnhance. mermaid` ，可以正常使用。
由于你提供的信息太少，我无法判断发生了什么问题。

**Reply by zhengchen-zzZ** [reply]
下图在typora中可以渲染：
![image-20241021234405327](https://gitee.com/kilomi/pic-bed/raw/master/img/202410212344661.png)
![image-20241021234348971](https://gitee.com/kilomi/pic-bed/raw/master/img/202410212343371.png)
但是在页面显示为：
![image-20241021234603940](https://gitee.com/kilomi/pic-bed/raw/master/img/202410212346277.png)
我不确定是否是我配置有遗漏，依赖和配置如下：
![image-20241021234740544](https://gitee.com/kilomi/pic-bed/raw/master/img/202410212347806.png)
![image-20241021234716798](https://gitee.com/kilomi/pic-bed/raw/master/img/202410212347140.png)
![image-20241021234701996](https://gitee.com/kilomi/pic-bed/raw/master/img/202410212347337.png)

**Reply by pengzhanbo** [reply]
配置名是 `markdownEnhance`，不是 `markdownenhance`， 大小写有问题。

**Reply by zhengchen-zzZ** [reply]
ok
